### DISCUSSION TITLE
Attach Multiple Reality models without detaching the active ones

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#3284] [Q&A] [answered]
Is there any way to attach a reality model without detaching the previously active one? I want the capability of turning on multiple models at the same time.. (just like in contextShare)

**Comment by pmconne** [comment, marked as answer]
Yes. If you show your current code, it may be easier to help. But [this sample](https://www.itwinjs.org/sample-showcase/?group=Viewer&sample=reality-data-sample&imodel=Exton+Campus) displays two reality models simultaneously.

**Reply by hanan-commits-work** [reply]
thank you I will look into it.
